src/components/Amalgamation/ForeignBusiness.vue

```diff
@@ -0,0 +1,191 @@
+import NestedSelect from '@/components/common/NestedSelect.vue'
+import { AmalgamatingBusinessIF } from '@/interfaces'
+import { AmlRoles, Location } from '@/enums'
+import { CanJurisdictions, IntlJurisdictions } from '@/list-data'
+
+@Component({
+  components: { NestedSelect }
+})
+export default class ForeignBusiness extends Vue {
+  // Refs
+  $refs!: {
+    foreignBusinessForm: any
+  }
+
+  // Local properties
+  jurisdiction = null
+  legalName = null
+  corpNumber = null
+  isCan = false
+
+  // Validation properties
+  jurisdictionErrorMessage = ''
+
+  get isValid (): boolean {
+    return (this.jurisdiction && this.legalName && this.corpNumber)
+  }
+
+  get rules (): Array<(v) => boolean | string> {
+    return [ v => !!v || 'Required.' ]
+  }
+
+  /** The jursidiction select options */
+  get jurisdictionOptions (): Array<any> {
+    const array = [] as Array<any>
+
+    // add in Canadian jurisdictions (not including BC)
+    array.push({ isHeader: true, group: 0, text: 'Canadian' })
+    CanJurisdictions
+      .filter(jur => jur.value !== Location.BC)
+      .forEach(jur => array.push({
+        group: 0,
+        text: jur.text,
+        value: jur.value,
+        separator: (jur.value === Location.FD)
+      }))
+
+    // add in International jurisdictions (not including CA)
+    array.push({ isHeader: true, group: 1, text: 'International' })
+    IntlJurisdictions
+      .filter(jur => jur.value !== Location.CA)
+      .forEach(jur => array.push({
+        group: 1,
+        text: jur.text,
+        value: jur.value,
+        separator: false
+      }))
+
+    return array
+  }
+
+  /** Called when Jurisdiction menu item is changed. */
+  onJurisdictionChange (jurisdiction: any): void {
+    this.jurisdiction = jurisdiction
+    this.isCan = jurisdiction.group === 0
+    this.jurisdictionErrorMessage = this.jurisdiction ? '' : 'Required.'
+  }
+
+  /** Add the amalgamating foreign business. */
+  addForeignBusiness (): void {
+    // Validate
+    this.validate()
+    if (!this.isValid) return
+
+    // Create the amalgamating foreign business object
+    const business = {
+      type: 'foreign',
+      role: AmlRoles.AMALGAMATING,
+      foreignJurisdiction: {
+        region: this.isCan ? this.jurisdiction.text : '',
+        country: this.isCan ? Location.CA : this.jurisdiction.value
+      },
+      legalName: this.legalName,
+      corpNumber: this.corpNumber
+    } as AmalgamatingBusinessIF
+
+    // Emit the amalgamating foreign business.
+    this.emitForeignBusiness(business)
+  }
+
+  /** Validate the inputs. */
+  validate (): void {
+    this.jurisdictionErrorMessage = this.jurisdiction ? '' : 'Required.'
+    this.$refs.foreignBusinessForm.validate()
+  }
+
+  @Emit('add:foreignBusiness')
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  private emitForeignBusiness (foreignBusiness: AmalgamatingBusinessIF): void {}
+
+  @Emit('cancel')
+  emitCancel (): boolean {
+    return true
```

Comment:
You probably don't have to emit an actual value. Just emitting the event should work. See how others are done.